addons/t9n/views/t9n_project_views.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0"?>
+<odoo>
+    <record id="t9n.project_action" model="ir.actions.act_window">
+        <field name="name">Projects</field>
+        <field name="res_model">t9n.project</field>
+        <field name="view_mode">tree,form</field>
+    </record>
+
+    <record id="t9n.project_view_form" model="ir.ui.view">
+        <field name="name">t9n.project.form</field>
+        <field name="model">t9n.project</field>
+        <field name="arch" type="xml">
+            <form>
+                <sheet>
+                    <h1>
+                        <field name="name"/>
+                    </h1>
+                    <notebook>
+                        <page string="Resources">
+                            <field name="resource_ids"/>
+                        </page>
+                        <page string="Description">
+                            <group>
+                                <group>
+                                    <field name="src_lang_id"/>
+                                    <field name="target_lang_ids"/>
```

Comment:
Might be useful to add in a `domain` parameter (https://www.odoo.com/documentation/master/developer/reference/backend/orm.html#odoo.fields.Many2one) for these fields so that you can't choose a target language that's the same as the source language and vice versa. But we can also just add in a check later on to ignore src = target lang cases in our business logic

Could also add in an onchange warning (https://www.odoo.com/documentation/master/developer/reference/backend/orm.html#odoo.api.onchange) if the user tries to change one of these fields and it overlaps with the other's (e.g. the source language is changed to a language in its target language)

These could both also be added in later on though since it's just an extra polish thing

Reply:
Done!